### PR TITLE
feat: added ability to destory plants

### PR DESCRIPTION
**Pull request by MyuddinKhatri**
Ref: [Asana](https://app.asana.com/0/1194641031921089/1188496226446181)

destroy_pant

### CODE REVIEW

**Review by neilLasrado — CHANGES_REQUESTED** [review]
Can you change the checkbox to a Select field with 2 options, Plant or Plant Batch?

File: erpnext/agriculture/doctype/destroyed_plant_log/destroyed_plant_log.json [hunk 1]
```diff
@@ -13,12 +15,13 @@
  ],
  "fields": [
   {
+   "allow_in_quick_entry": 1,
+   "depends_on": "eval: doc.type == \"Plant Batch\"",
    "fieldname": "plant_batch",
    "fieldtype": "Link",
```
Comment (hrwX) [inline erpnext/agriculture/doctype/destroyed_plant_log/destroyed_plant_log.json]: Use Dynamic Link for this

**Review by hrwX — CHANGES_REQUESTED** [review]

**Comment by hrwX** [comment]
@MyuddinKhatri check if this is a duplicate https://github.com/Bloomstack/erpnext/pull/633

**Comment by MyuddinKhatri** [comment]
> @MyuddinKhatri check if this is a duplicate #633

@hrwX I checked #633 it is same as what i did in this PR, as well as #633 has some more enhancements so can i close this PR?

**Pull request closed without merging**